### ISSUE TITLE
export png/svg bug: extra margins in result, besides in firefox transparent areas turn black

### ISSUE THREAD

**Issue by imanzaker** [excalidraw/excalidraw#8317]
through "copy to clipboard" and "export image" the PNG and SVG result files get a 10px extra margin on every four sides that doesn't make sense at all. 
In Firefox all transparent areas turn black. [A video of this bug](https://go.screenpal.com/watch/cZje2LVPSJu)

**Comment by eelpik** [comment]
I tried setting the export padding to 0 in the ImageExportDialog.tsx component, and some of the drawing's edges get clipped, so maybe that's why it's there?
![clipped](https://github.com/user-attachments/assets/3b9126cf-bc53-466a-b47c-463bc6767808)
It does feel strange to apply the padding to images, though, where this clipping doesn't seem to be an issue.

**Comment by dwelle** [comment]
> I tried setting the export padding to 0 in the ImageExportDialog.tsx component, and some of the drawing's edges get clipped, so maybe that's why it's there?

Correct, that was one of the main reasons, although we're still likely going to allow customizing padding. The question is whether `0` padding should use the bounding box of the shape, or the exact pixel bounding box (which will differ based on the random sloppiness).

**Comment by imanzaker** [comment]
The exact pixel of the shape makes more sense for zero. Whoever wants to have more part of the edge, is gotta set the padding to 1 or greater. Putting a padding size input field on the dialog box would be more helpful for getting the appropriate look of the exported image.
